queries/dune_v2/period_slippage.sql

```diff
@@ -266,6 +259,17 @@ incoming_and_outgoing as (
     ) as _
     order by block_time
 )
+-- These batches involve a token who do not emit standard transfer events.
+,excluded_batches as (
```

Comment:
Yes. Without reordering we do include some of the batches with those tokens since they contain internalized transfers. Since the same problem of missing transfers holds for internalized trades we need to filter after those tables.